api/app/controllers/spree/api/v2/storefront/account/addresses_controller.rb

```diff
@@ -16,6 +16,14 @@ def update
               render_result(result)
             end
 
+            def destroy
```

Comment:
@pmilewski please take a look at https://github.com/spree/spree/blob/master/api/app/controllers/spree/api/v2/platform/resource_controller.rb#L36 - for destroy we need to return empty body with 204 status code